src/main/java/gui/GuiSetting.java

```diff
@@ -0,0 +1,20 @@
+package gui;
+
+import io.github.bonigarcia.wdm.WebDriverManager;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.chrome.ChromeDriver;
+import utilities.Browser;
+
+public class GuiSetting {
+
+    /**
+     * Sets Browser type.
+     *
+     * @param browser which we are tested
+     * @return type of browser
+     */
+    public WebDriver setBrowser(final Browser browser) {
```

Comment:
This method only set the driver for Chrome, maybe, should be a more generic method, for other drivers too

Reply:
solved